geospaas/nansat_ingestor/managers.py

```diff
@@ -90,7 +92,12 @@ def get_or_create(self,
         for name in default_char_fields:
             if name not in n_metadata:
                 warnings.warn('%s is not provided in Nansat metadata!' % name)
-                options[name] = default_char_fields[name]()
+                #prevent the existing values from overwriten action by those of default values
+                if len(Dataset.objects.filter(entry_id=entry_id))>0:
+                    existing_ds = Dataset.objects.get(entry_id=entry_id)
```

Comment:
Move this block before the loop over name in default_char_fields
```python
        try:
            existing_ds = Dataset.objects.get(entry_id=entry_id)
        except (Dataset.DoesNotExist, Dataset.MultipleObjectsReturned):
            existing_ds = None
```
Then add this inside loop:
```suggestion
                #prevent overwriting of existing values by defaults
                if existing_ds:
```